src/operators/expand-support.ts

```diff
@@ -41,24 +58,31 @@ export class ExpandSubscriber<T, R> extends OuterSubscriber<T, R> {
     }
 
     const index = this.index++;
-    if (this.active < this.concurrent) {
+    if (this.active < this._concurrent) {
       destination.next(value);
       let result = tryCatch(this.project)(value, index);
       if (result === errorObject) {
         destination.error(result.e);
+      } else if (!this.scheduler) {
+        this._innerSub(result, value, index);
       } else {
-        if (result._isScalar) {
-          this._next(result.value);
-        } else {
-          this.active++;
-          this.add(subscribeToResult<T, R>(this, result, value, index));
-        }
+        const state = { subscriber: this, result, value, index };
+        this.add(this.scheduler.schedule(disptch, 0, state));
       }
     } else {
       this.buffer.push(value);
     }
   }
 
+  _innerSub(result: any, value: any, index: number): void {
```

Comment:
Yes, I was suspecting that it means private. Ideally `innerSub` should be private. Currently, it isn't because `dispatch` cannot be an instance method (AFAIK).